### PR TITLE
Carsus atomic merge

### PR DESCRIPTION
**Pull request by ftsamis**
This is a rebase to current upstream master of #604

### CODE REVIEW

File: tardis/io/atomic.py [hunk 1]
```diff
@@ -196,13 +196,13 @@ def __init__(self, atom_data, ionization_data, levels=None, lines=None,
 
         # SET ATTRIBUTES
 
-        self.atom_data = atom_data
-        self.ionization_data = ionization_data
+        self.atom_data = atom_masses
```
Comment (wkerzendorf) [inline tardis/io/atomic.py]: make the attribute `atom_masses` as well.

File: tardis/io/atomic.py [hunk 2]
```diff
@@ -196,13 +196,13 @@ def __init__(self, atom_data, ionization_data, levels=None, lines=None,
...
+        self.ionization_data = ionization_energies
```
Comment (wkerzendorf) [inline tardis/io/atomic.py]: same goes for here.

File: tardis/io/atomic.py [hunk 3]
```diff
@@ -181,12 +181,12 @@ def __init__(self, atom_data, ionization_data, levels=None, lines=None,
         # We have to use constants.u because astropy uses different values for the unit u and the constant.
         # This is changed in later versions of astropy (the value of constants.u is used in all cases)
         if u.u.cgs == const.u.cgs:
-            atom_data.loc[:, "mass"] = Quantity(atom_data["mass"].values, "u").cgs
```
Comment (yeganer) [inline tardis/io/atomic.py]: change 'u' to 'g'

File: tardis/io/atomic.py [hunk 4]
```diff
@@ -181,12 +181,12 @@ def __init__(self, atom_data, ionization_data, levels=None, lines=None,
...
+            atom_masses.loc[:, "mass"] = Quantity(atom_masses["mass"].values, "u").cgs
         else:
-            atom_data.loc[:, "mass"] = atom_data["mass"].values * const.u.cgs
+            atom_masses.loc[:, "mass"] = atom_masses["mass"].values * const.u.cgs
```
Comment (yeganer) [inline tardis/io/atomic.py]: change 'u' to 'g' here, too

**Comment by unoebauer** [comment]
Closing this one - reopen if necessary, @wkerzendorf, @yeganer

**Pull request closed without merging**